org.eclipse.jface.text/src/org/eclipse/jface/text/link/LinkedModeUI.java

```diff
@@ -290,7 +290,7 @@ public interface IExitPolicy {
 		 * @param model the linked mode model
 		 * @param event the document event
 		 * @return valid exit flags or <code>null</code> if no special action should be taken
-		 * @since 3.22
+		 * @since 3.23
```

Comment:
This mustn't be modified.

Reply:
yes I understand the risk, was to let the  check pass and mean to revert back